plugins/jobs/girder_jobs/web_client/webpack.helper.js

```diff
@@ -0,0 +1,15 @@
+module.exports = function (config) {
+    config.module.rules.push({
+        resource: {
+            test: /\.js$/,
+            include: [/node_modules\//]
+        },
```

Comment:
This will add babel to all js files in node_modules globally.  If that's the intent, you might as well just add this rule to the top level webpack config.

Reply:
On my machine this appears to only affect the build of the jobs plugin target, not any other plugins or the core build.

Reply:
Yeah, I guess it generate a separate webpack config object for each plugin.  This is fine then.